packages/test-commerce-sdk-react/app/pages/useShopperLoginHelper.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import {ShopperLoginHelpers, useShopperLoginHelper} from 'commerce-sdk-react'
+import Json from '../components/Json'
+
+const UseShopperLoginHelper = () => {
+    const loginGuestUser = useShopperLoginHelper(ShopperLoginHelpers.LoginGuestUser)
```

Comment:
Dumb question. What happens if you don't pass anything into the useShopperLoginHelper hook?

Reply:
the hook throws an error `throw new Error('Unknown ShopperLogin helper.')`

for typescript projects, the IDE looks like 
![Screen Shot 2022-09-11 at 11 56 21 PM](https://user-images.githubusercontent.com/10948652/189591777-fb22dfc6-f30d-4488-9c03-4a2bbe77e002.png)